src/main/java/org/team1540/robot2024/subsystems/indexer/IndexerIO.java

```diff
@@ -0,0 +1,33 @@
+package org.team1540.robot2024.subsystems.indexer;
+
+import org.littletonrobotics.junction.AutoLog;
+
+public interface IndexerIO {
+
+    @AutoLog
+    class IndexerIOInputs {
+        public double intakeVoltage;
+        public double intakeCurrentAmps;
+        public double intakeVelocityRPM;
+        public double feederVoltage;
+        public double feederCurrentAmps;
+        public double feederVelocityRPM;
+        public boolean noteInIntake = false;
+        public double setpointRPM;
+        public double feederPositionError;
+    }
+
+    /**
+     * Updates the set of loggable inputs.
+     */
+    default void updateInputs(IndexerIOInputs inputs) {}
+
+    default void setIntakeVoltage(double volts) {}
+
+    default void setFeederVoltage(double volts) {}
+
+    default void setFeederVelocity(double velocity) {}
+
+    default void configurePID(double p, double i, double d) {}
```

Comment:
This method is never implemented, you might want to do that. Also, this should probably be called `configureFeederPID` since the intake isn't running a control loop